### PR TITLE
fix: changes how env var is read

### PR DESCRIPTION
**Pull request by telpirion**
This should hopefully reduce the amount of bucket leakage we see in the test project.

### CODE REVIEW

**Comment by ivanmkc** [comment]
@telpirion I think the check in of ai-platform-unified/notebooks/official/automl/prediction_results/output/prediction-e2e-text-classification-model-20210715152855-2021-07-15T22:59:08.992688Z/predictions_00001.jsonl is unintentional.

File: ai-platform-unified/notebooks/official/automl/automl-text-classification.ipynb [hunk 1]
```diff
@@ -913,7 +913,7 @@
       },
       "outputs": [],
       "source": [
-        "if os.getenv(\"IS_TESTING\") is True:\n",
+        "if os.getenv(\"IS_TESTING\") == \"1\":\n",
```
Comment (ivanmkc) [inline ai-platform-unified/notebooks/official/automl/automl-text-classification.ipynb:916]: Feel free to use

```
if os.getenv(\"IS_TESTING\"):
```

Which is less explicit but is more standardized.

File: ai-platform-unified/notebooks/official/automl/prediction_results/output/prediction-e2e-text-classification-model-20210715152855-2021-07-15T22:59:08.992688Z/predictions_00001.jsonl [hunk 2]
```diff
@@ -0,0 +1,2 @@
+{"instance":{"content":"gs://automl-text-notebook-20210715225703/input_1.txt","mimeType":"text/plain"},"prediction":{"ids":["4212216951381950464","7670981465202491392","465222061409697792","6518059960595644416","1906373942168256512","5076908079837085696","8823902969809338368"],"displayNames":["affection","achievement","enjoy_the_moment","bonding","nature","leisure","exercise"],"confidences":[0.97810465,0.013285442,0.0052765086,0.0023137697,7.675774E-4,2.3890579E-4,1.3138302E-5]}}
```
Comment (ivanmkc) [inline ai-platform-unified/notebooks/official/automl/prediction_results/output/prediction-e2e-text-classification-model-20210715152855-2021-07-15T22:59:08.992688Z/predictions_00001.jsonl:1]: Delete this file

File: ai-platform-unified/notebooks/official/automl/automl-text-classification.ipynb (hunk 1 above)
Comment (telpirion) [reply ai-platform-unified/notebooks/official/automl/automl-text-classification.ipynb]: Done.

File: ai-platform-unified/notebooks/official/automl/prediction_results/output/prediction-e2e-text-classification-model-20210715152855-2021-07-15T22:59:08.992688Z/predictions_00001.jsonl (hunk 2 above)
Comment (telpirion) [reply ai-platform-unified/notebooks/official/automl/prediction_results/output/prediction-e2e-text-classification-model-20210715152855-2021-07-15T22:59:08.992688Z/predictions_00001.jsonl]: Done.

**Review by ivanmkc — APPROVED** [review]
LGTM

**Pull request merged**